README.md

```diff
@@ -54,6 +54,7 @@ Finally, I'm sure everyone who reads this list has one thing they want to add. P
 - [grunt-photobox](https://github.com/stefanjudis/grunt-photobox) - Plugin to prevent your project of broken layout via screenshot photo sessions of your site
 - [vrtest](https://github.com/nathanmarks/vrtest) - A JavaScript library for running visual regression tests on your components cross browser via selenium
 - [Happo](https://github.com/Galooshi/happo) - Visual diffing in CI for user interfaces
+- [reg-cli](https://github.com/bokuweb/reg-cli) - Visual regression test tool
```

Comment:
> Visual regression test tool

Could you please be a little more specific!
What makes reg-cli special? What are the target system? Such things...